### ISSUE TITLE
Feature: Year-Overview

### ISSUE THREAD

**Issue by manuel3108** [beyonk-adventures/svelte-datepicker#7]
It would be quite cool to have the possibility to select between years in the calendar.

It could be similar to the month overview we currently have, but with the years in it.

How could it be displayed:
* If you are on the month overview, another click on the month name could open the year overview
* We visually separate the month and the year in the top a bit more, and a click on the month would open the month overview as it currently does. A click on the year could open the year overview.

**Comment by antony** [comment]
Yep - I think this would be an excellent convenience feature. Great idea.
